scripts/lint/ourlint.js

```diff
@@ -170,7 +170,7 @@ const TESTS = {
 
     const invalidUrls = [];
     for (const [isSourceUrl, url] of allUrlFields) {
-      const $url = new global.URL(url);
+      const $url = new globalThis.URL(url);
```

Comment:
Silly question but do we need to prefix the URL constructor with global-anything in the first place?

Reply:
Yes. It's a good practice and good for other runtime/compiler like Bun, Deno, ncc, SWC, etc.